pkg/network/proxy.go

```diff
@@ -49,5 +78,76 @@ func validateKubeProxy(conf *operv1.NetworkSpec) []error {
 		}
 	}
 
+	// Don't allow ports to be overridden
+	if _, ok := p.ProxyArguments["metrics-port"]; ok {
+		out = append(out, errors.Errorf("kube-proxy --metrics-port cannot be overridden"))
```

Comment:
should allow it to be "overridden" to the default value, right?

Reply:
Do you think that's necessary? I'm not against it, but it's not obviously needed to me.

Reply:
I'm worried some existing cluster might be doing that. That might be paranoid...

Reply:
If they're doing that, then they're going to see many, many failures in the system Prometheus, since it will be failing to scrape. So I'm not concerned.

Reply:
sorry, when I said "default", I meant *our* default, not the default default. In other words, if someone is explicitly setting it to 9101, we should allow that

Reply:
*shrug* ok.